mrtt-ui/src/library/usePopulateQuestionFormWithInitialValues.js

```diff
@@ -0,0 +1,36 @@
+import axios from 'axios'
+import { toast } from 'react-toastify'
+import { useEffect } from 'react'
+
+import formatApiAnswersForForm from './formatApiAnswersForForm'
+import language from '../language'
+
+const usePopulateQuestionFormWithInitialValues = ({
```

Comment:
for lack of a better name....

Reply:
useInitializeForm?

Reply:
I'd also specify to which type of form it supports in the naming. I'd also cut the same path over a certain amount of time. See xkcd git commit naming comic.

Reply:
We have forms it wont work on in the app so i need to indicate that its not generic. The struggle I am having is what to call the 'question forms'. Site forms? (but the other forms do pertain to a site, but the api is different so they dont get formatted like the question forms) 

We could always rely on copy paste for form init too, lol. Sometimes wet code is better if you cant name the thing. I think this is passable. @julia-rom what do we name this?

Reply:
which are the question forms ? do you mean the non-site creation forms? I am confused by the name too and it's function @mmnoo

Reply:
if so, can we just have `initializeFormdata` and `initializeSiteData`?